setup.py

```diff
@@ -31,8 +31,9 @@
         # long as they have a version of pyfoobar equal to or greater than 1.x
         # and less than 2.x installed.
         'flask>=1.0.2',
-        'marshmallow>=2.15.3,<3.0',
-        'marshmallow-annotations>=2.4.0,<3.0'
+        'marshmallow>=2.15.3,<=3.6',
+        ('git+https://www.github.com/hilearn/marshmallow-annotations.git@a7a2dc96932430369bd'
```

Comment:
do we need `(` here?

Reply:
I believe so because it is 1 logn string that I had to break up without having a newline

Reply:
I believe there would be an indentation error without the parenthesis